.github/workflows/pull-lifecycle-mgr-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: pull-lifecycle-mgr-lint
+on:
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+          cache: false
```

Comment:
why not cache enabled?

Reply:
No thoughts given to that, just taken from the golangci-lint action sample.
Just read a bit about it, for the cache to work, we need to make sure that the checkout step is executed before, which is the case here.
Will try with enable cache (default for v4 setup-go)